app/src/main/java/com/github/se/polyfit/ui/screen/OverviewScreen.kt

```diff
@@ -316,6 +325,55 @@ fun OverviewScreen(
                   }
                 }
           }
+          item {
+            OutlinedCard(
+                modifier =
+                    Modifier.align(Alignment.Center)
+                        .padding(top = 10.dp)
+                        .size(width = 350.dp, height = 300.dp)
+                        .testTag("Graph Card")
+                        .clickable { navController.navigate(Route.Graph) },
```

Comment:
To keep our navigation consistent, we should use the `navigation` variable defined on line 206, and create a function to navigate to graph

```suggestion
                        .clickable { navigation.navigateToGraph() },
```